addons/spreadsheet/static/src/pivot/pivot_model.js

```diff
@@ -371,38 +353,32 @@ export class OdooPivotModel extends PivotModel {
     /**
      * Check if the given field is used as col group by
      */
-    _isCol(field) {
-        return this.metaData.fullColGroupBys
-            .map(this.parseGroupField)
-            .map(({ field }) => field.name)
-            .includes(field.name);
+    _isCol(dimensionName) {
+        return this.metaData.fullColGroupBys.includes(dimensionName);
```

Comment:
does this work if the same date field is in the cols and rows with with a different granularity ?

Reply:
Yes, the dimensionName is nameWithGranularity. Will update the name of the param